### ISSUE TITLE
Missing image based on sha

### ISSUE THREAD

**Issue by hippee-lee** [SeleniumHQ/docker-selenium#965]
## 💥 Regression Report

A clear and concise description of what the regression is.

There was a script that was running a specific implementation of the image so that we could lock onto a specific version of chrome. The command run the image with the sha: e.g 

```
docker run --rm --name=clarity_chrome -d -p 4444:4444 selenium/standalone-chrome@sha256:b899f16b6d963600ef6da8a8dd49e311146033ed66cb5af71eccb78ab378e19a
```

The script that runs this has not changed in ~2 years but it stopped working today. 
It seem like the commit for that verion of chrome jsut disappeared. I'm sorry but I'm not a docker expert so I may be confused with how the git sha relates to the docker sha on dockerhub. 

## Last working version

Worked up to version: b899f16b6d963600ef6da8a8dd49e311146033ed66cb5af71eccb78ab378e19a

Stopped working in version:
@latest / HEAD? 
## To Reproduce

this used to run:
```
docker run --rm --name=clarity_chrome -d -p 4444:4444 selenium/standalone-chrome@sha256:b899f16b6d963600ef6da8a8dd49e311146033ed66cb5af71eccb78ab378e19a
```

## Expected behavior
It should start the docker instance. 

## Test script reproducing this issue (when applicable)
The script that has been running this command is [here](https://github.com/vmware/clarity/blob/master/scripts/docker-cdc.js#L70).

## Environment

OS: 
Docker-Selenium image version: 
Docker version: 
![Screen Shot 2019-09-11 at 9 27 27 AM](https://user-images.githubusercontent.com/433692/64716038-770bc400-d476-11e9-9ea2-da6160ede1ee.png)

Docker-Compose version (if applicable):
Exact Docker command to start the containers (if using docker-compose, provide 
the docker-compose file as well):

**Comment by diemol** [comment]
@hippee-lee I was able to pull the image with the sha you mentioned, so I cannot reproduce the issue.
Anyway, I notice that this is a 2-year-old image, please try to use a recent tag to avoid these type of situations.

**Comment by hippee-lee** [comment]
Thanks for checking @diemol. I'm now able to pull it as well. I'm not sure what changed but will keep investigating the script I'm using 

wrt - the image, I would love to use a newer one but we found issues with the gemini visual regression testing tool where different instances of Chrome cause different results when the images are generated. This was causing issues with our CI pipeline because sometimes images could be generated with one version of chrome and tested with a different version of chrome and there seems to be small visual differences that trigger the test to fail. So, I locked it onto one container rather than always having to update reference images each time a new image was created. We are moving away from gemini though so it won't be using that image for much longer.
